src/quarterback/useSnapshotStore.ts

```diff
@@ -83,7 +83,6 @@ export const useSnapshotStore = create(
             currentDocument.manuscriptID,
             {
               docID: currentDocument.manuscriptID,
-              snapshot: docJson,
```

Comment:
@mbartenev-atypon this has been removed based on @hhusban comment

@hhusban
this PR is generating a build error because ISaveSnapshotRequest type definition does not include the property "snapshot: docJson",

This was done by design, creating a snapshot no longer needs the document, it will be fetched from the Database.